modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -734,7 +733,7 @@ public Response getMetadataCacheDetails() {
         }
 
         // Only include `assignment` property if there is a definitive assignment, either to a bucket or not
-        if (assignment.getStatus() != EXPERIMENT_EXPIRED) {
+        if (assignment.getStatus().isDefinitiveAssignment()) {
```

Comment:
@AndreaSuckro I take back my comment, I think technically this IF condition is NOT backward compatible but practically there are no bucket labels in case of NON-definitive assignment so this change looks good 👍